dockerfiles/mlrun-api/Dockerfile

```diff
@@ -15,6 +15,11 @@ ARG MLRUN_PYTHON_VERSION=3.7.13
 
 FROM quay.io/mlrun/python:${MLRUN_PYTHON_VERSION}-slim
 
+
+ARG UID=1000
+# this creates both user and group with the same id
+RUN useradd -u $UID mlrun
```

Comment:
Don't we want to prevent no home dir? Running with -M to prevent the needlessly home dir creation